lib/src/spinner/Spinner.tsx

```diff
@@ -110,14 +109,14 @@ const DxcSpinner = ({
   );
 };
 
-const determinatedValue = (props, strokeDashArray) => {
+const determinatedValue = (value, strokeDashArray) => {
```

Comment:
Sorry not to tell you this before, but can you add the `SpinnerPropsType["value"]` and `number` types to both parameters?